Day-II-Calculator/calculator/src/index.css

```diff
@@ -2,4 +2,5 @@ body {
   margin: 0;
   padding: 0;
   font-family: sans-serif;
+  width: 57.5%;
```

Comment:
You want to remove the `width:57.5%` it's forcing the body to be smaller than it needs to be.